docs/samples/Microsoft.ML.GenAI.Samples/Phi3Mini/Utils.cs

```diff
@@ -20,7 +20,7 @@ public static ICausalLMPipeline<Tokenizer, Phi3ForCasualLM> LoadPhi3Mini4KFromFo
         string weightFolder,
         string configName = "config.json",
         string device = "cuda",
-        int modelSizeOnCudaInGB = 16,
```

Comment:
Did you want to change this to 55 GB? I don't know anyone with a local GPU that size.

Reply:
This parameter is used to set how much size of model weight should be loaded to GPU instead of how much size does GPU has. So the number here could be larger or smaller than the actual GPU size. And if the number is larger than actual GPU size, an OOM will be thrown